### PR TITLE
Fix Datasource named ${DS_PROMETHEUS} was not found

### PR DESCRIPTION
**Pull request by MalloZup**
Lately by deploying dashboard this was failing.

I'm setting to wip since I dunno if it is the proper fix I didn't verified in detail so let it PR open and see tomorrow.

### CODE REVIEW

File: dashboards/grafana-sap-netweaver.json [hunk 1]
```diff
@@ -1,46 +1,4 @@
 {
-  "__inputs": [
-    {
-      "name": "DS_PROMETHEUS",
```
Comment (MalloZup) [inline dashboards/grafana-sap-netweaver.json:4]: strange enough that the other dashboard seems to have this

**Comment by MalloZup** [comment]
it seems a well known issue see: https://github.com/kubernetes/ingress-nginx/issues/4190

**Comment by stefanotorresi** [comment]
I don't quite understand this diff: it looks like the variable was there but it's being removed here!

**Comment by MalloZup** [comment]
@stefanotorresi yep right. This is what grafana dumped me when I added a variable and saved the dashboard..

**Comment by MalloZup** [comment]
before merging this I will build a pkg and run it on a server.

**Comment by MalloZup** [comment]
related to https://github.com/grafana/grafana/issues/10786

**Comment by stefanotorresi** [comment]
@MalloZup how did you dump this json? Did you use the "share dashboard" function with the "export for sharing externally" option on? We shouldn't be deleting the first blocks and `id` should be `null`.

**Comment by stefanotorresi** [comment]
I'm also thinking that we should add the explicit grafana 7.1.5 version in the spec file `Recommends` field, so that we track where this was tested on.

**Comment by MalloZup** [comment]
https://github.com/SUSE/sap_host_exporter/pull/53

**Pull request closed without merging**